Lib/logging/__init__.py

```diff
@@ -1244,6 +1244,17 @@ def _fixupChildren(self, ph, alogger):
                 alogger.parent = c.parent
                 c.parent = alogger
 
+    def _clear_cache(self):
+        """
+        Clear the cache for all loggers in loggerDict
+        Called when level changes are made
+        """
+
+        _acquireLock()
```

Comment:
Isn't it needed to keep another thread from changing loggerDict or the cache dictionaries?